kola-denylist.yaml

```diff
@@ -27,3 +27,8 @@
   tracker: https://github.com/coreos/coreos-assembler/issues/2725
   arches:
   - ppc64le
+- pattern: ext.config.shared.chrony.dhcp-propagation
+  tracker: https://github.com/openshift/os/issues/1352
+  arches:
+  - s390x
+  - ppc64le
```

Comment:
maybe make this a snooze so we'll get reminded to check back in on it?

Reply:
Right, I forgot about the snooze part